src/js/yeldaWebchatInjector.js

```diff
@@ -158,25 +159,37 @@ class YeldaChat {
         return
       }
 
-      const customImage =  settings.data.image && settings.data.image.url
+      const isVoiceFirstUI = settings.data.hasOwnProperty('isVoiceFirstUI') || false
+      const customImage = settings.data.image && settings.data.image.url
       const hasCustomStyle = settings.data.hasOwnProperty('isDefaultStyle') && !settings.data.isDefaultStyle
 
-      if (!hasCustomStyle || !customImage) {
-        this.webChatContainer.appendChild(this.assistantImage)
-        return
-      }
+      /**
+       * when isVoiceFirstUI is true, instead of adding the assistant image open the chat here,
+       * website-chat will take care rendering the voice first UI
+       */
+      if (!data.isDemo && isVoiceFirstUI) {
```

Comment:
why (!data.isDemo) ? isDemo has the right to have isVoiceFirstUI
Can you explain ?

Reply:
I thought we don't show the voice first ui now. that's why, now I have removed it

Reply:
Default value is false, so nobody is supposed to see it yet (excepting if they turn it on)